modelina-website/src/components/playground/options/JavaGeneratorOptions.tsx

```diff
@@ -187,8 +234,15 @@ class JavaGeneratorOptions extends React.Component<
             />
           </label>
         </li>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Include Overwrite toString Support :">
+            <p>
+            In Java, the toString() method is used to provide a string representation of an object. The default implementation in the Object class returns a string consisting of the class name, an "@" symbol, and the hexadecimal representation of the object's hash code.
+            <br/><br/>
+            By enabling the option to overwrite toString() support, you can define your own implementation of the toString() method for a class. This allows you to provide a more meaningful and customized string representation of the object based on its properties.
```

Comment:
This is very generic wording about what `toString` is, however it does not explain what it does to the code generation.

Reply:
fixed!! :tada: